CODE-OWNERS

```diff
@@ -0,0 +1,35 @@
+areas:
+  all:
+    approvers:
+    - travisn
+    - galexrt
+    - jbw976
+  common:
```

Comment:
What is "common"? These should somehow be documented.

Reply:
It's intended to represent the rook.io common types, specs, logic, etc.  I have added a comment block to the top of this file to document each area.

Reply:
The general idea makes sense, but I'm still not clear on exactly what areas of the code constitute "common". Maybe this would be more clear once there is configuration for whatever bot enforces these rules.